### ISSUE TITLE
May i ask a relevant issue？ what's the best accuracy that monodepth estimation can achieve from a distance of 100 m

### ISSUE THREAD

**Issue by vicentowang** [FangGet/PackNet-SFM-PyTorch#4]
I need a monodepth estimation to achieve at least 88%  accuracy from 80 - 100m , may it possible for now ?

**Comment by FangGet** [comment]
We never did evaluation for distance between 80 - 100m. For now, depth estimation is based on geometry constraint, with further distance a big error as nature, I think it's hard for now to get 88% accuracy.

**Comment by vicentowang** [comment]
hi is me again , may i ask what the difference between original and improved dataset at Table3 ,which should be concerned ？


------------------&nbsp;原始邮件&nbsp;------------------
发件人:&nbsp;"Fangget"<notifications@github.com&gt;;
发送时间:&nbsp;2020年2月25日(星期二) 下午2:11
收件人:&nbsp;"FangGet/PackNet-SFM-PyTorch"<PackNet-SFM-PyTorch@noreply.github.com&gt;;
抄送:&nbsp;"渚薰岚"<849529792@qq.com&gt;;"Author"<author@noreply.github.com&gt;;
主题:&nbsp;Re: [FangGet/PackNet-SFM-PyTorch] May i ask a relevant issue？ what's the best accuracy that monodepth estimation can achieve from a distance of 100 m (#4)




Closed #4.

**Comment by FangGet** [comment]
Not understand what you say, please make it more clearly.

**Comment by vicentowang** [comment]
”Table 3: Quantitative performance comparison of PackNet-SfM on the KITTI dataset for distances up to 80m. For Abs Rel, Sq Rel, RMSE and RMSElog lower is better, and for δ < 1.25, δ < 1.252 and δ < 1.253 higher is better. In the Datasetcolumn,CS+KreferstopretrainingonCityScapes(CS)andﬁne-tuningonKITTI(K).Mreferstomethodsthattrain using monocular (M) images, and M+v refers to added velocity weak supervision (v), as shown in Section 3.2. ‡ indicates ImageNet[10]pretraining. Original uses raw depth maps from[14] for evaluation,and Improved uses annotated depth maps from [42]. At test-time, all monocular methods (M) scale estimated depths with median ground-truth LiDAR information. Velocity-scaled (M+v) and supervised (D) methods are not scaled in such way, since they are already metrically accurate.“


I means i do not understand that "Original uses raw depth maps from[14] for evaluation,and Improved uses annotateddepth maps from [42]."
what is the difference between raw depth maps&nbsp; and annotated depth maps ? it may refer to&nbsp;Sparsity Invariant CNNs.





------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Fangget"<notifications@github.com&gt;; 
发送时间: 2020年3月3日(星期二) 晚上11:03
收件人: "FangGet/PackNet-SFM-PyTorch"<PackNet-SFM-PyTorch@noreply.github.com&gt;; 
抄送: "渚薰岚"<849529792@qq.com&gt;; "Author"<author@noreply.github.com&gt;; 
主题: Re: [FangGet/PackNet-SFM-PyTorch] May i ask a relevant issue？ what&#39;s the best accuracy that monodepth estimation can achieve from a distance of 100 m (#4)




Not understand what you say, please make it more clearly.
 
&mdash;

**Comment by FangGet** [comment]
Sorry, the version of "packnet-sfm" paper I got has no information about your metioned "annotated depth maps".

why not just refer to the given linked paper[14] and [42] for more information?
